generation/um/xmllite/generate.rsp

```diff
@@ -0,0 +1,22 @@
+@../../settings.rsp
+@../../remap.rsp
+--exclude
+IID_IXmlReader
+IID_IXmlWriter
+IID_IXmlResolver
+_DtdProcessing_Last
```

Comment:
The `_Last` enum values are generally not excluded from bindings as they are still useful for various checks/asserts.